core/arch/opnd.h

```diff
@@ -2844,6 +2844,7 @@ bool
 reg_is_stolen(reg_id_t reg);
 
 /* internal version */
+
```

Comment:
Please revert this extra line -- this makes this comment confusingly separated from its function.

Reply:
Done.